packages/devtools_app/test/vm_service_private_test.dart

```diff
@@ -1,43 +0,0 @@
-// Copyright 2020 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-import 'package:devtools_app/src/service/service_manager.dart';
-import 'package:devtools_app/src/service/vm_service_wrapper.dart';
-import 'package:devtools_app/src/shared/globals.dart';
-import 'package:devtools_test/devtools_test.dart';
-import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
-import 'package:vm_service/vm_service.dart';
-
-void main() {
-  test('Ensure private RPCs can only be enabled with VM Developer Mode enabled',
-      () async {
-    final service = createMockVmServiceWrapperWithDefaults();
-
-    when(service.trackFuture(any, any)).thenAnswer(
-      (invocation) => invocation.positionalArguments[1],
-    );
-    when(service.callMethod(argThat(equals('_collectAllGarbage')))).thenAnswer(
-      (_) => Future.value(Success()),
-    );
-
-    final fakeServiceManager = FakeServiceManager(service: service);
-    setGlobal(ServiceConnectionManager, fakeServiceManager);
-    VmServicePrivate.enablePrivateRpcs = false;
-
-    try {
```

Comment:
Although this spec is a bit odd, I believe it still serves to tests that this exception is thrown
https://github.com/flutter/devtools/blob/374f37db70ab18acf1055807c4f59d14e85d24ce/packages/devtools_app/lib/src/service/vm_service_wrapper.dart#L909

So I think it is still providing value, although it may benefit us to rewrite it to be a bit cleaner in the future

Reply:
This test does not touch VmServiceWrapper. It works with MockVmServiceWrapper instead.

Reply:
I chatted with @polina-c about this and she has mentioned that the test doesn't really work as depicted and isn't providing value any more. Her expert opinion is good enough for me :D